### PR TITLE
Feature/issue 35

### LINKED ISSUE #35: Update Setup instructions in README
- Replace some of the steps with the parts that the `acme init` command covers
- add instructions on installing the `hygen` dependency
- add instructions on installing `@babel/preset-typescript` dependency for storybook's webpack to compile the project's scss/js
- check that the npm commands have the correct directories/other details

### PR DESCRIPTION
**Pull request by davekenison**
closes #35 

## Details

- What I did
Documentation update to match current state of application

- How to test
- Does this change require an update to the documentation?

Include answers to these questions and any other relevant details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,32 +44,19 @@ This is the format for the expected config file. As of now, all fields are requi
...
-                        |-- title_1     // Fancy Button
-                        |-- button_1
-```
+## AEM Content
 
-You can reference the [acme sample project](https://github.com/hoodoo-digital/acme-storybook-sample-project) for details.
+`acme` parses the rendered HTML of the Core Components example pages to generate stories based on those components. `acme` uses the Title component before each example to name each story and looks for the Demo container component to know where each individual component begins and ends. The AEM Core Component examples must be installed on the instance configured in the `baseURL` parameter of the `acme.settings.json` file.
 
-## Command line reference
+## Command Line Reference
```
Comment (madoni) [inline README.md:59]: @davekenison Can you update the cli output too?

File: README.md [hunk 2]
```diff
@@ -41,32 +44,19 @@ This is the format for the expected config file. As of now, all fields are requi
...
-                    |-- container
-                        |-- title       // Simple Button
-                        |-- button
-                        |-- title_1     // Fancy Button
-                        |-- button_1
-```
+## AEM Content
 
-You can reference the [acme sample project](https://github.com/hoodoo-digital/acme-storybook-sample-project) for details.
+`acme` parses the rendered HTML of the Core Components example pages to generate stories based on those components. `acme` uses the Title component before each example to name each story and looks for the Demo container component to know where each individual component begins and ends. The AEM Core Component examples must be installed on the instance configured in the `baseURL` parameter of the `acme.settings.json` file.
```
Comment (madoni) [inline README.md:57]: Can we link to some documentation on how to install Core Components Examples?

File: README.md [hunk 3]
```diff
@@ -138,6 +128,8 @@ Steps to get up and running with `acme` on a fresh AEM archetype project.
     npm install storybook-aem-style-system -D
     npm install storybook-addon-xd-designs -D
     npm install @hoodoo/acme -D
+    npm install hygen -D
+    npm install @babel/preset-typescript -D
```
Comment (madoni) [inline README.md:132]: I think we can condense this to one line:
```suggestion
    npm install -D storybook-aem-style-system storybook-addon-xd-designs @hoodoo/acme hygen @babel/preset-typescript
```

File: README.md [hunk 4]
```diff
@@ -22,17 +22,20 @@ or
 export DEBUG=acme:*
 ```
 
-## Configuration file
+## Configuration File
 
-This is the format for the expected config file. As of now, all fields are required.
+This is the format for the expected config file. As of now, all fields are required. Convention is to name this file `acme.settings.json`. You can use the `acme init` command to trigger a prompt that will automatically create this file.
```
[the remark this answers is not in the document]
Comment (davekenison) [reply README.md:27]: good idea, I'll add that

File: README.md (hunk 1 above)
Comment (davekenison) [reply README.md:59]: yep, working on it

File: README.md (hunk 2 above)
Comment (davekenison) [reply README.md:57]: Yep, I'll add that

File: README.md (hunk 3 above)
Comment (davekenison) [reply README.md:132]: looks good

File: README.md [hunk 5]
```diff
@@ -41,46 +44,46 @@ This is the format for the expected config file. As of now, all fields are requi
...
-                    |-- container
-                        |-- title       // Simple Button
-                        |-- button
-                        |-- title_1     // Fancy Button
-                        |-- button_1
-```
+`acme` parses the rendered HTML of the Core Components example pages to generate stories based on those components. `acme` uses the Title component before each example to name each story and looks for the Demo container component to know where each individual component begins and ends. The AEM Core Component examples must be installed on the instance configured in the `baseURL` parameter of the `acme.settings.json` file.
 
-You can reference the [acme sample project](https://github.com/hoodoo-digital/acme-storybook-sample-project) for details.
+Instructions for installing the AEM Core Components can be found here: [AEM WCM Core Components](https://github.com/adobe/aem-core-wcm-components)
```
Comment (madoni) [inline README.md:59]: @davekenison  should we specify the `core.wcm.components.examples-x.x.x.zip` should be installed? -  https://github.com/adobe/aem-core-wcm-components/releases/download/core.wcm.components.reactor-2.8.0/core.wcm.components.examples-2.8.0.zip

Comment (davekenison) [reply README.md:59]: I added the name of the specific package that they need to have installed (`core.wcm.components.examples`)--that package is included in the core components release so I think that should be clear enough. The examples are installed by default with a fresh archetype project.

File: README.md [hunk 6]
```diff
@@ -41,46 +44,46 @@ This is the format for the expected config file. As of now, all fields are requi
...
-  acme create  Create stories from AEM content
+  acme init [file]  Generate the configuration file
+  acme pull         Pull content from AEM
+  acme create       Create stories from AEM content
+
+Options:
+  --help     Show help                                                 [boolean]
+  --version  Show version number                                       [boolean]
+```
+```bash
```
Comment (madoni) [inline README.md:74]:
```suggestion
### `init`
```bash
```

File: README.md [hunk 7]
```diff
@@ -41,46 +44,46 @@ This is the format for the expected config file. As of now, all fields are requi
...
+Generate the configuration file
+
+Positionals:
+  file  Configuration file path         [string] [default: "acme.settings.json"]
 
 Options:
   --help     Show help                                                 [boolean]
   --version  Show version number                                       [boolean]
 ```
 ```bash
```
Comment (madoni) [inline README.md:86]:
```suggestion
### `pull`
```bash
```

File: README.md [hunk 8]
```diff
@@ -93,7 +96,6 @@ Options:
   --config           Path to JSON config file                         [required]
 ```
 ```bash
```
Comment (madoni) [inline README.md:98]:
```suggestion
### `create`
```bash
```

**Pull request merged**